akka-http/src/main/scala/akka/http/scaladsl/server/directives/MiscDirectives.scala

```diff
@@ -71,6 +72,27 @@ trait MiscDirectives {
     BasicDirectives.extractRequest.map { request ⇒
       LanguageNegotiator(request.headers).pickLanguage(first :: List(more: _*)) getOrElse first
     }
+
+  /**
+   * Rejects all requests which request entity size exceeds given limit. Limit given
```

Comment:
It does not `reject`. Rejection is something specific in Akka HTTP: http://doc.akka.io/docs/akka/2.4.7/scala/http/routing-dsl/rejections.html#rejections-scala

See docs here https://github.com/akka/akka/blob/master/akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala#L111

It fails the stream if it exceeds the limit with an `EntityStreamSizeException`.